packages/@lwc/ssr-runtime/src/__tests__/stubs.spec.ts

```diff
@@ -0,0 +1,9 @@
+import * as runtime from '../index';
+import * as stubs from '../stubs';
+
+describe('no stubs are unused', () => {
+    test.for(Object.entries(stubs))('%s is used', ([key, value]) => {
+        expect(runtime).toHaveProperty(key);
+        expect(runtime[key as keyof typeof runtime]).toBe(value);
+    });
+});
```

Comment:
Very nice! Too bad our linters aren't smart enough to notice this for us.

Reply:
I expected JS itself to complain about duplicate exports, but I guess it's more flexible with `*`?